scripts/reset_iks_api_key.sh

```diff
@@ -22,13 +23,25 @@ if [[ -z "${RESOURCE_GROUP_ID}" ]]; then
     exit 1
 fi
 
+if [[ -z "${PRIVATE_ENV}" ]]; then
+    PRIVATE_ENV=false
+fi
+
 # Login to ibmcloud with cli
 attempts=1
-until ibmcloud login -q -r "${REGION}" -g "${RESOURCE_GROUP_ID}" || [ $attempts -ge 3 ]; do
-    attempts=$((attempts+1))
-    echo "Error logging in to IBM Cloud CLI..." >&2
-    sleep 5
-done
+if [ "$PRIVATE_ENV" ]; then
```

Comment:
given that the difference is only the cloud endpoint if private env is true or false, wouldn't be better to set the cloud endpoint value according to the private env value and don't repeat the entire code in if and else statement?